### PR TITLE
Clean up ENABLE_V2 feature flag

### PR DESCRIPTION
**Pull request by aeolianeth**
Closes https://github.com/jbx-protocol/juice-interface/issues/833

### CODE REVIEW

**Comment by aeolianeth** [comment]
yo @wraeth-eth , up for a review on this?

File: src/components/Landing/Footer.tsx [hunk 1]
```diff
@@ -61,9 +60,6 @@ export default function Footer() {
         {link('Twitter', 'https://twitter.com/juiceboxETH')}
         {link('Privacy Policy', '/#/privacy')}
       </div>
-      <div style={{ display: 'flex', margin: 'auto' }}>
```
Comment (wraeth-eth) [inline src/components/Landing/Footer.tsx:64]: Was the switch only supposed to be transitory for rinkeby testing?

Comment (wraeth-eth) [reply src/components/Landing/Footer.tsx:64]: I think I answered my own question by scrolling down - yes :P

File: src/components/Landing/index.tsx [hunk 2]
```diff
@@ -169,29 +161,16 @@ export default function Landing() {
...
-                        size="large"
-                        onClick={() => {
-                          window.fathom?.trackGoal('IIYVJKNC', 0)
-                          scrollToCreate()
-                        }}
-                      >
+                    <Link
+                      to={'/create'}
+                      onClick={() => {
+                        window.fathom?.trackGoal('IIYVJKNC', 0)
```
Comment (wraeth-eth) [inline src/components/Landing/index.tsx:167]: unrelated to this PR: should we turn the `IIYVJKNC` into a constant?

File: src/components/Landing/index.tsx [hunk 3]
```diff
@@ -169,29 +161,16 @@ export default function Landing() {
...
+                      }}
+                    >
+                      <Button type="primary" size="large">
                         <Trans>Design your project</Trans>
```
Comment (wraeth-eth) [inline src/components/Landing/index.tsx:171]: I just realised that `Design your project` is inconsistent with the Projects tab (`Create Project`). Might be fine though

**Review by wraeth-eth — APPROVED** [review]

**Comment by wraeth-eth** [comment]
Another thing to note - when travelling to `/v1/create` we see this:

https://user-images.githubusercontent.com/104132113/168013993-21412032-5356-49a3-b777-fc44b0aea90a.mov

File: src/components/Landing/index.tsx (hunk 2 above)
Comment (aeolianeth) [reply src/components/Landing/index.tsx:167]: Yeah def, this tracking stuff needs to be centralized into a lib/service somewhere. https://github.com/jbx-protocol/juice-interface/issues/986

File: src/components/Landing/index.tsx (hunk 3 above)
Comment (aeolianeth) [reply src/components/Landing/index.tsx:171]: Yeah thats a good point, I think it's fine tbh. Kinda contextually different. I'm also proposing to change this to `Launch your project` on the homepage, which probably doesn't make as much sense as `Create project` on the Projects page.

Idk, I don't feel too strongly though.

Comment (wraeth-eth) [reply src/components/Landing/index.tsx:171]: Neither do I - happy to leave. just thought i'd raise to have a discussion about it

**Pull request merged**